ext/writable/P0441.md

```diff
@@ -0,0 +1,1593 @@
+---
+pagetitle: 'Ranges: Merging Writable and MoveWritable'
+title: 'Ranges: Merging Writable and MoveWritable'
+...
+
+# Introduction
+<!-- This paper presents several design changes to N4560, the working paper for the Technical Specification for C++ Extensions for Ranges (the "Ranges TS"). Each change will be presented herein in a separate section that motivates the change and describes the design problem that change is meant to address. Technical specifications are relatively brief, since the complete specification and wording of all the changes is in N4569, a document that speculatively integrates these changes completely with the wording of N4560. This process is "unusual" in that the proposal authors failed to grok the process changes resulting from incorporating the evolving proposal into a TS working paper.
```

Comment:
Need to add an actuall reference here to the Ranges TS; otherwise, it won't show up in the "Reference" section.